events/scripts/past-events.js

```diff
@@ -120,11 +125,27 @@ upcomeEventHtmlCode += `
 
```

Comment:
In the code above this line (line 108) add an option to choose from mainEvent array's 0th element or 1st element depeneding upon the event commencement date. Such that once an event expires, fallback data comes automatically. at the moment, it seems you have hard coded to use the mainEvent[0] data.